pkg/cmd/kill/kill.go

```diff
@@ -0,0 +1,112 @@
+/*
```

Comment:
This should be in `pkg/cmd/container/kill.go`, as the canonical form of `nerdctl kill` is `nerdctl container kill`

Reply:
Yes, please